control-services/bin/setup.sh

```diff
@@ -26,6 +26,7 @@ sudo apt-get upgrade -y
 sudo rm -rf /usr/local/go
 wget https://go.dev/dl/go1.17.7.linux-amd64.tar.gz -P /tmp/
 sudo tar -C /usr/local/ -xzf /tmp/go1.17.7.linux-amd64.tar.gz
+export PATH=$PATH:/usr/local/go/bin
```

Comment:
Does this need to happen at line 29?

At the bottom:
`echo 'export PATH="$PATH:/usr/local/go/bin:/home/control/go/bin' | sudo tee -a /etc/profile`

Reply:
I didn't catch that in the bottom, but yes it needs to happen before calling "go version". So perhaps just moving what's from the bottom and putting it on line 29. I can refractor that today.

Reply:
There is another possibility of removing the "check go version" around line 30, or placing that after the path has been exported so it doesn't completely fail out of the setup.sh script.

Another clarifying question, why are we adding `home/control/go/bin` to `$PATH`? when the `go/bin` is in `/usr/local/`?

Reply:
Added Comment of overall changes. Removed the line `go version`.